assignments/arrays.js

```diff
@@ -63,32 +63,45 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let filteredArray = invetory.filter((car)) => {
+    if (car.id === 33){
+        return true;
+    }
+    return false;
+})
+console.log('Car 33 is a '+ 2011 + ', '+ Jeep + ', '+ Wrangler)
```

Comment:
This is hard coding the answer. In most cases the data is coming in from an API so you don't know what it is until it is received. Not to mention it is constantly changing whether the car is sold, another added to the list etc.